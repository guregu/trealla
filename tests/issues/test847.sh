#!/bin/sh

TMP=tmp.out
TMPPL=tmp.pl

trap "rm -f $TMPPL" EXIT

cat >$TMPPL <<EOF
EOF

$TPL -f $TMPPL >$TMP <<EOF
length(L,3).
halt.
EOF

