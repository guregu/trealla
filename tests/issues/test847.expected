   L = [_A,_B,_C].
